app/src/hooks/useGetCounter.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from 'react'
+
+export default function useGetCounter(seconds: number, isCountingUp: boolean) {
+  const [counterContent, setCounterContent] = useState(isCountingUp ? 0 : seconds)
```

Comment:
The start of the counter (now at 0) should not be hardcoded.
0 should be the default value, but we should be able to initialize the counter at diferent values.
For example, with a pomodoro, it would be a value of 25x60 (1500).

Also, I'd rename this to `counterValue`, to make it more clear what it contains.